eng/xplat-job.yml

```diff
@@ -29,7 +30,7 @@ jobs:
     timeoutInMinutes: ${{ parameters.timeoutInMinutes }}
 
     # Send telemetry for official builds
-    enableTelemetry: ${{ and(eq(variables['System.TeamProject'], 'internal'), ne(variables['Build.Reason'], 'PullRequest')) }}
+    enableTelemetry: true
```

Comment:
I am excited about this change.  I think it will really help with visibility into what is happening in the pr.